### PR TITLE
email information for users requesting app access ok

### PR DESCRIPTION
**Pull request by SamirTouira**
issue: #5806 

Do not forget to deploy the backend-functions after this merge

### CODE REVIEW

File: apps/backend-functions/src/notification.ts [hunk 1]
```diff
@@ -443,3 +449,11 @@ async function sendOfferCreatedConfirmation(recipient: User, notification: Notif
   const template = offerCreatedConfirmationEmail(org, notification.bucket, recipient);
   await sendMailFromTemplate(template, app, unsubscribeId);
 }
+
+/** User receive a notification and an email to confirm his request access has been sent*/
+async function requestAccessAppEmail(recipient: User, notification: NotificationDocument) {
```
Comment (Mussuranae) [inline apps/backend-functions/src/notification.ts:454]: Same.

File: apps/backend-functions/src/notification.ts [hunk 2]
```diff
@@ -198,6 +199,11 @@ export async function onNotificationCreate(snap: FirebaseFirestore.DocumentSnaps
           .then(() => notification.email.isSent = true)
           .catch(e => notification.email.error = e.message);
         break;
+      case "userRequestAppAccess":
+        await requestAccessAppEmail(recipient, notification)
```
Comment (Mussuranae) [inline apps/backend-functions/src/notification.ts:203]: Change to `requestAppAccessEmail`.

**Review by Mussuranae — CHANGES_REQUESTED** [review]
Almost :D

**Comment by SamirTouira** [comment]
I changed that 👍

**Review by Mussuranae — APPROVED** [review]
LGTM :+1:

**Pull request merged**